### DISCUSSION TITLE
v7.1.0

### DISCUSSION THREAD

**Discussion by antonmedv** [deployphp/deployer#3427] [General]
- Fixed PHP [8.1.14](https://www.php.net/downloads.php#v8.1.14) compatibilities. 
- Added Magento2 artifact deployment (#3317)
- Added pub/static/_cache as shared_dir (#3343)
- Added NewRelic endpoint to config and used it when posting to api (#3374)
- Fix: use include path when importing YAML files (#3377)
- Laravel recipe: Add horizon publish command (#3425)
- Added CraftCMS recipe (#3405) 
- Added Cloudflare API Token support for auth, domain optional if zone_id provided (#3406)
- Added --yes flag to Caddy gpg command (#3407)
- Laravel recipe: Skip database tasks if .env is missing (#3359)
- Fix: #3319 - Receiving variables db_user and db_name (#3351)
- Fix: escape metainfo to prevent bash syntax errors (#3328)
- Additional writable modes (#3316)
- Fixed typo for SSH_KNOWN_HOSTS in CI/CD docs (#3310)
- Fix: only delete symlink on broken release
- Improved error message: show next free release

<hr /><em>This discussion was created from the release <a href='https://github.com/deployphp/deployer/releases/tag/v7.1.0'>v7.1.0</a>.</em>

**Comment by mfickers** [comment]
Was PHP 7.4 support dropped intentionally? Our staging deployment pipeline just failed:
```
Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 7.4.16. in phar:///root/.composer/vendor/deployer/deployer/dep/vendor/composer/platform_check.php on line 24
```

I've noticed that the PHP requirement has been dropped from the composer.json, but the patch notes don't mention anything.

**Reply by antonmedv** [reply]
Looks like composer update did updated to incompatible deps. Will release a new version with php7 support.

**Reply by antonmedv** [reply]
Fixed.

**Reply by mfickers** [reply]
Just upgraded to 7.1.1 and can confirm that the issue is fixed. Thank you for the quick fix!
